src/app/pages/administration/login/login.jsx

```diff
@@ -22,16 +26,24 @@ function ChplLoginPage(props) {
 
   if (ssoIsOn) {
     return (
-      <ChplCognitoLogin
-        dispatch={handleLogin}
-      />
+      <Box py={'4vh'} bgcolor={palette.background}>
```

Comment:
```suggestion
      <Box py="4vh" bgcolor={palette.background}>
```

Please try this. Usually we only need the `{ something here }` stuff when that "something here" needs to read variables or do logic. If it's simple strings they can usually just be done with the `"something"` construct